extras/userdoc/md/documentation/installation.md

```diff
@@ -450,34 +499,3 @@ please try making NEST as follows:
     env MACOSX_DEPLOYMENT_TARGET=10.5 make
 
 Thanks to Harold Gutch for this hint.
```

Comment:
I removed the entire section, since this addressed problems that occurred with with OSX 10.5 very long ago.